assets/src/dashboard/components/popover-menu/index.js

```diff
@@ -40,7 +40,8 @@ export const Menu = styled.ul`
   box-shadow: 0px 4px 14px rgba(0, 0, 0, 0.25);
   display: flex;
   flex-direction: column;
-  margin: 20px 0;
+  margin: ${({ framelessButton }) => (framelessButton ? '0' : '20px 0')};
+  min-width: 210px;
```

Comment:
<img width="396" alt="Screen Shot 2020-03-23 at 3 39 37 PM" src="https://user-images.githubusercontent.com/1738349/77463525-be82ed80-6dd3-11ea-9f46-bb8280665e53.png">

The idea here is to ensure the left-hand text margins match up.

Reply:
thanks!